src/components/EnlargedPreview/index.tsx

```diff
@@ -115,10 +130,9 @@ class EnlargedPreview extends Component {
   }
 }
 
-export default compose(
-  withNavigationDisabledState,
-  withNavigationDisableAction,
-  withFullScreenState,
-  withFullScreenAction,
-  localised
-)(EnlargedPreview)
+export default withNavigationDisabledState(
+  withNavigationDisableAction(
+    // @ts-ignore wait for NavigationBar
```

Comment:
Navigation bar is now merged @nulrich

Reply:
I updated `withNavigationDisabledState` and `withNavigationDisableAction`. I'm using `useSelector` and `useDispatch`. I couldn't find a way to get valid types with `connect`.